lib/natalie/compiler/pass0c.rb

```diff
@@ -24,7 +25,62 @@ def process_iter(exp)
         )
       end
 
+      def process_defined(exp, block_with_args=nil)
+        @in_defined_statement = true
+        exp = exp.new(*exp.map {|sub_exp| sub_exp.is_a?(Sexp) ? process(sub_exp) : sub_exp})
+        @in_defined_statement = false
+        return exp
+      end
```

Comment:
This might fail on nested `defined?` calls, but who would do such a thing

Reply:
yeah, sounds unlikely, but just in case, it might be worth adding a comment :D